core/src/main/java/com/graphhopper/util/Parameters.java

```diff
@@ -191,9 +191,9 @@
      */
     public static final class DETAILS {
 
-        private static final String DETAILS_PREFIX = "details.";
```

Comment:
> Do I understand you right, that I should revert the change then

Yes, the variable should be named DETAILS_PREFIX

> Not that this would be an issue, just wanted to mention it

Yes, I know. I do not see an issue with this as we can always change UI debug parameters IMO